yarn.lock

```diff
@@ -8659,15 +8659,14 @@ loader-utils@1.1.0:
     emojis-list "^2.0.0"
     json5 "^0.5.0"
 
-loader-utils@^0.2.5:
```

Comment:
dropping this will fix https://github.com/getsentry/sentry/security/dependabot/117